### PR TITLE
Reintroduce "Allowing manual Call On from dispatcher window" - https://trello.com/c/I19VVKlz/450-manual-call-on-from-dispatcher-window

### PR DESCRIPTION
**Pull request by cesarBLG**
For some reason #155 has vanished, so I create this PR with the feature.

### CODE REVIEW

File: Source/Orts.Simulation/Simulation/Signalling/Signals.cs [hunk 1]
```diff
@@ -12116,6 +12118,7 @@ public bool CheckTimingTrigger(int reqTiming, string dumpfile)
 
         public bool TrainHasCallOn(bool allowOnNonePlatform, bool allowAdvancedSignal, string dumpfile)
         {
+            CallOnEnabled = true;
```
Comment (YoRyan) [inline Source/Orts.Simulation/Simulation/Signalling/Signals.cs:12121]: Here's a difference between this code and the one presented in #155 - in the previous PR, you set this property in SIGSCRfile.cs, but now, you've made it always true. Can you explain this change?

**Comment by YoRyan** [comment]
Sorry you lost your PR, and thanks for creating a new one. For future reference, don't merge into the `unstable` branch. That one's automatically managed by the Code Bot. Always use `master`.

File: Source/Orts.Simulation/Simulation/Signalling/Signals.cs (hunk 1 above)
Comment (cesarBLG) [reply Source/Orts.Simulation/Simulation/Signalling/Signals.cs:12121]: Rob created two additional callon functions between the previous PR and this one, and all of them call this function. Setting this variable within TrainHasCallOn() is better than having to set it separately for each function.

The aim of this variable is to show the "Enable Call On" option in the signal's context menu only for signals which support this feature.

**Comment by cesarBLG** [comment]
I think it was my first PR, and I wasn't aware of the code bot. Now that I know about it, I always compare against the `master` branch.

File: Source/Orts.Simulation/Simulation/Signalling/Signals.cs (hunk 1 above)
Comment (YoRyan) [reply Source/Orts.Simulation/Simulation/Signalling/Signals.cs:12121]: The purpose of the function is to "Test if train has call-on set," which strongly implies that there shouldn't be any side effects. Therefore, I think it's best to set this variable back in SIGSCRfile.cs, regardless of the code duplication.

Comment (cesarBLG) [reply Source/Orts.Simulation/Simulation/Signalling/Signals.cs]: I moved it back to SIGSCRfile.cs

**Review by YoRyan — APPROVED** [review]
Thanks! Approving and merging.

**Pull request merged**